Weather/Weather/WViewController/ViewController.m

```diff
@@ -0,0 +1,158 @@
+//
+//  ViewController.m
+//  Weather
+//
+//  Created by Sangwook Wi on 5/2/17.
+//  Copyright © 2017 Sangwook Wi. All rights reserved.
+//
+
+#import "ViewController.h"
+#import "WModel.h"
+#import "WCenter.h"
+#import "WTableViewCell.h"
+
+@interface ViewController() {
+    WCenter *cInstance;
+    CGRect originalsize;
+    NSArray *dayModel;
+}
+
+@property(strong, nonatomic) NSMutableArray *daysCell;
+@property(strong, nonatomic) NSMutableArray *highTempCell;
+@property(strong, nonatomic) NSMutableArray *lowTempCell;
+@property(strong, nonatomic) NSMutableArray *weatherCell;
```

Comment:
I was wondering when these arrays would come into play but it looks like you aren't using these anymore? Don't forget to delete code you no longer need in your final PRs!